elasticsearch-evolution-core/src/test/java/com/senacor/elasticsearch/evolution/core/ElasticsearchEvolutionIT.java

```diff
@@ -47,18 +46,12 @@
 
     private HistoryRepository historyRepository;
 
-    @ParameterizedTest
+    @ParameterizedTest(name = "esVersion: {0}")
```

Comment:
should be reflecting what the test is doing, like because it replaces the method name. Suggestion: `should migrate without errors on esVersion {0}`

Reply:
I had it like this before but then i thought its redundant since the method name is displayed and then the name is used for the invocation. like:
migrate_OK
 - esVersion 7.5.2
 ...
 -esVersion 7.10.0
I think during the maven build this is ignored  and we only get something like:

`[ERROR] Failures: 
[ERROR]   ElasticsearchEvolutionIT.migrate_OK:60 
Multiple Failures (1 failure)
-- failure 1 --
[# of historyIndex entries and all are successful] 
Expected size:<8> but was:<7> in:
<[MigrationScriptProtocol{version=1, indexName='null', description='createTemplateWithIndexMapping', scriptName='V001.00__createTemplateWithIndexMapping.http', checksum=513517984, executionTimestamp=2020-11-15T10:54:38.426378+01:00, executionRuntimeInMillis=116, success=true, locked=false},
    MigrationScriptProtocol{version=1.1, indexName='null', description='addDocument', scriptName='V001.01__addDocument.http', checksum=164793414, executionTimestamp=2020-11-15T10:54:38.678987+01:00, executionRuntimeInMillis=227, success=true, locked=false},
    MigrationScriptProtocol{version=2, indexName='null', description='extendIndexMappingViaTemplate', scriptName='V002.00__extendIndexMappingViaTemplate.http', checksum=-819319052, executionTimestamp=2020-11-15T10:54:38.780903+01:00, executionRuntimeInMillis=80, success=true, locked=false},
    MigrationScriptProtocol{version=2.1, indexName='null', description='updateMappingOfExistingIndices', scriptName='V002.01__updateMappingOfExistingIndices.http', checksum=787213132, executionTimestamp=2020-11-15T10:54:38.897617+01:00, executionRuntimeInMillis=96, success=true, locked=false},
    MigrationScriptProtocol{version=2.2, indexName='null', description='moveFieldToSearchable', scriptName='V002.02__moveFieldToSearchable.http', checksum=-644953840, executionTimestamp=2020-11-15T10:54:39.580007+01:00, executionRuntimeInMillis=659, success=true, locked=false},
    MigrationScriptProtocol{version=3, indexName='null', description='addDocumentWithNewStructure', scriptName='V003.00__addDocumentWithNewStructure.Http', checksum=1643942673, executionTimestamp=2020-11-15T10:54:39.687193+01:00, executionRuntimeInMillis=63, success=true, locked=false},
    MigrationScriptProtocol{version=3.1, indexName='null', description='addDocumentWithNewStructureToNewIndex', scriptName='V003.01__addDocumentWithNewStructureToNewIndex.http', checksum=1130329556, executionTimestamp=2020-11-15T10:54:39.940837+01:00, executionRuntimeInMillis=236, success=true, locked=false}]>
at ElasticsearchEvolutionIT.lambda$migrate_OK$0(ElasticsearchEvolutionIT.java:66)
[ERROR]   ElasticsearchEvolutionIT.migrate_OK:60 
Multiple Failures (1 failure)
-- failure 1 --
[# of historyIndex entries and all are successful] 
Expected size:<8> but was:<7> in:
<[MigrationScriptProtocol{version=1, indexName='null', description='createTemplateWithIndexMapping', scriptName='V001.00__createTemplateWithIndexMapping.http', checksum=513517984, executionTimestamp=2020-11-15T10:54:42.971975+01:00, executionRuntimeInMillis=76, success=true, locked=false},
    MigrationScriptProtocol{version=1.1, indexName='null', description='addDocument', scriptName='V001.01__addDocument.http', checksum=164793414, executionTimestamp=2020-11-15T10:54:43.315803+01:00, executionRuntimeInMillis=299, success=true, locked=false},
    MigrationScriptProtocol{version=2, indexName='null', description='extendIndexMappingViaTemplate', scriptName='V002.00__extendIndexMappingViaTemplate.http', checksum=-819319052, executionTimestamp=2020-11-15T10:54:43.416751+01:00, executionRuntimeInMillis=78, success=true, locked=false},
    MigrationScriptProtocol{version=2.1, indexName='null', description='updateMappingOfExistingIndices', scriptName='V002.01__updateMappingOfExistingIndices.http', checksum=787213132, executionTimestamp=2020-11-15T10:54:43.532949+01:00, executionRuntimeInMillis=87, success=true, locked=false},
    MigrationScriptProtocol{version=2.2, indexName='null', description='moveFieldToSearchable', scriptName='V002.02__moveFieldToSearchable.http', checksum=-644953840, executionTimestamp=2020-11-15T10:54:44.184914+01:00, executionRuntimeInMillis=628, success=true, locked=false},
    MigrationScriptProtocol{version=3, indexName='null', description='addDocumentWithNewStructure', scriptName='V003.00__addDocumentWithNewStructure.Http', checksum=1643942673, executionTimestamp=2020-11-15T10:54:44.299391+01:00, executionRuntimeInMillis=94, success=true, locked=false},
    MigrationScriptProtocol{version=3.1, indexName='null', description='addDocumentWithNewStructureToNewIndex', scriptName='V003.01__addDocumentWithNewStructureToNewIndex.http', checksum=1130329556, executionTimestamp=2020-11-15T10:54:44.554323+01:00, executionRuntimeInMillis=225, success=true, locked=false}]>
at ElasticsearchEvolutionIT.lambda$migrate_OK$0(ElasticsearchEvolutionIT.java:66)
[ERROR]   ElasticsearchEvolutionIT.migrate_OK:60 
Multiple Failures (1 failure)
-- failure 1 --
[# of historyIndex entries and all are successful] 
Expected size:<8> but was:<7> in:
<[MigrationScriptProtocol{version=1, indexName='null', description='createTemplateWithIndexMapping', scriptName='V001.00__createTemplateWithIndexMapping.http', checksum=513517984, executionTimestamp=2020-11-15T10:54:46.534086+01:00, executionRuntimeInMillis=97, success=true, locked=false},
    MigrationScriptProtocol{version=1.1, indexName='null', description='addDocument', scriptName='V001.01__addDocument.http', checksum=164793414, executionTimestamp=2020-11-15T10:54:46.841305+01:00, executionRuntimeInMillis=278, success=true, locked=false},
    MigrationScriptProtocol{version=2, indexName='null', description='extendIndexMappingViaTemplate', scriptName='V002.00__extendIndexMappingViaTemplate.http', checksum=-819319052, executionTimestamp=2020-11-15T10:54:46.953267+01:00, executionRuntimeInMillis=93, success=true, locked=false},
    MigrationScriptProtocol{version=2.1, indexName='null', description='updateMappingOfExistingIndices', scriptName='V002.01__updateMappingOfExistingIndices.http', checksum=787213132, executionTimestamp=2020-11-15T10:54:47.064453+01:00, executionRuntimeInMillis=93, success=true, locked=false},
    MigrationScriptProtocol{version=2.2, indexName='null', description='moveFieldToSearchable', scriptName='V002.02__moveFieldToSearchable.http', checksum=-644953840, executionTimestamp=2020-11-15T10:54:47.712849+01:00, executionRuntimeInMillis=634, success=true, locked=false},
    MigrationScriptProtocol{version=3, indexName='null', description='addDocumentWithNewStructure', scriptName='V003.00__addDocumentWithNewStructure.Http', checksum=1643942673, executionTimestamp=2020-11-15T10:54:47.824334+01:00, executionRuntimeInMillis=89, success=true, locked=false},
    MigrationScriptProtocol{version=3.1, indexName='null', description='addDocumentWithNewStructureToNewIndex', scriptName='V003.01__addDocumentWithNewStructureToNewIndex.http', checksum=1130329556, executionTimestamp=2020-11-15T10:54:48.093043+01:00, executionRuntimeInMillis=249, success=true, locked=false}]>
at ElasticsearchEvolutionIT.lambda$migrate_OK$0(ElasticsearchEvolutionIT.java:66)
[ERROR]   ElasticsearchEvolutionIT.migrate_OK:60 
Multiple Failures (1 failure)
-- failure 1 --
[# of historyIndex entries and all are successful] 
Expected size:<8> but was:<7> in:
<[MigrationScriptProtocol{version=1, indexName='null', description='createTemplateWithIndexMapping', scriptName='V001.00__createTemplateWithIndexMapping.http', checksum=513517984, executionTimestamp=2020-11-15T10:54:49.898143+01:00, executionRuntimeInMillis=84, success=true, locked=false},
    MigrationScriptProtocol{version=1.1, indexName='null', description='addDocument', scriptName='V001.01__addDocument.http', checksum=164793414, executionTimestamp=2020-11-15T10:54:50.174860+01:00, executionRuntimeInMillis=249, success=true, locked=false},
    MigrationScriptProtocol{version=2, indexName='null', description='extendIndexMappingViaTemplate', scriptName='V002.00__extendIndexMappingViaTemplate.http', checksum=-819319052, executionTimestamp=2020-11-15T10:54:50.299695+01:00, executionRuntimeInMillis=73, success=true, locked=false},
    MigrationScriptProtocol{version=2.1, indexName='null', description='updateMappingOfExistingIndices', scriptName='V002.01__updateMappingOfExistingIndices.http', checksum=787213132, executionTimestamp=2020-11-15T10:54:50.428590+01:00, executionRuntimeInMillis=111, success=true, locked=false},
    MigrationScriptProtocol{version=2.2, indexName='null', description='moveFieldToSearchable', scriptName='V002.02__moveFieldToSearchable.http', checksum=-644953840, executionTimestamp=2020-11-15T10:54:50.966782+01:00, executionRuntimeInMillis=523, success=true, locked=false},
    MigrationScriptProtocol{version=3, indexName='null', description='addDocumentWithNewStructure', scriptName='V003.00__addDocumentWithNewStructure.Http', checksum=1643942673, executionTimestamp=2020-11-15T10:54:51.093155+01:00, executionRuntimeInMillis=98, success=true, locked=false},
    MigrationScriptProtocol{version=3.1, indexName='null', description='addDocumentWithNewStructureToNewIndex', scriptName='V003.01__addDocumentWithNewStructureToNewIndex.http', checksum=1130329556, executionTimestamp=2020-11-15T10:54:51.407081+01:00, executionRuntimeInMillis=291, success=true, locked=false}]>
at ElasticsearchEvolutionIT.lambda$migrate_OK$0(ElasticsearchEvolutionIT.java:66)
[ERROR]   ElasticsearchEvolutionIT.migrate_OK:60 
Multiple Failures (1 failure)
-- failure 1 --
[# of historyIndex entries and all are successful] 
Expected size:<8> but was:<7> in:
<[MigrationScriptProtocol{version=1, indexName='null', description='createTemplateWithIndexMapping', scriptName='V001.00__createTemplateWithIndexMapping.http', checksum=513517984, executionTimestamp=2020-11-15T10:54:53.490199+01:00, executionRuntimeInMillis=101, success=true, locked=false},
    MigrationScriptProtocol{version=1.1, indexName='null', description='addDocument', scriptName='V001.01__addDocument.http', checksum=164793414, executionTimestamp=2020-11-15T10:54:53.797571+01:00, executionRuntimeInMillis=287, success=true, locked=false},
    MigrationScriptProtocol{version=2, indexName='null', description='extendIndexMappingViaTemplate', scriptName='V002.00__extendIndexMappingViaTemplate.http', checksum=-819319052, executionTimestamp=2020-11-15T10:54:53.947195+01:00, executionRuntimeInMillis=129, success=true, locked=false},
    MigrationScriptProtocol{version=2.1, indexName='null', description='updateMappingOfExistingIndices', scriptName='V002.01__updateMappingOfExistingIndices.http', checksum=787213132, executionTimestamp=2020-11-15T10:54:54.093761+01:00, executionRuntimeInMillis=127, success=true, locked=false},
    MigrationScriptProtocol{version=2.2, indexName='null', description='moveFieldToSearchable', scriptName='V002.02__moveFieldToSearchable.http', checksum=-644953840, executionTimestamp=2020-11-15T10:54:54.801661+01:00, executionRuntimeInMillis=682, success=true, locked=false},
    MigrationScriptProtocol{version=3, indexName='null', description='addDocumentWithNewStructure', scriptName='V003.00__addDocumentWithNewStructure.Http', checksum=1643942673, executionTimestamp=2020-11-15T10:54:54.927269+01:00, executionRuntimeInMillis=104, success=true, locked=false},
    MigrationScriptProtocol{version=3.1, indexName='null', description='addDocumentWithNewStructureToNewIndex', scriptName='V003.01__addDocumentWithNewStructureToNewIndex.http', checksum=1130329556, executionTimestamp=2020-11-15T10:54:55.152721+01:00, executionRuntimeInMillis=208, success=true, locked=false}]>
at ElasticsearchEvolutionIT.lambda$migrate_OK$0(ElasticsearchEvolutionIT.java:66)
[ERROR]   ElasticsearchEvolutionIT.migrate_OK:60 
Multiple Failures (1 failure)
-- failure 1 --
[# of historyIndex entries and all are successful] 
Expected size:<8> but was:<7> in:
<[MigrationScriptProtocol{version=1, indexName='null', description='createTemplateWithIndexMapping', scriptName='V001.00__createTemplateWithIndexMapping.http', checksum=513517984, executionTimestamp=2020-11-15T10:54:56.983114+01:00, executionRuntimeInMillis=83, success=true, locked=false},
    MigrationScriptProtocol{version=1.1, indexName='null', description='addDocument', scriptName='V001.01__addDocument.http', checksum=164793414, executionTimestamp=2020-11-15T10:54:57.321897+01:00, executionRuntimeInMillis=320, success=true, locked=false},
    MigrationScriptProtocol{version=2, indexName='null', description='extendIndexMappingViaTemplate', scriptName='V002.00__extendIndexMappingViaTemplate.http', checksum=-819319052, executionTimestamp=2020-11-15T10:54:57.452350+01:00, executionRuntimeInMillis=101, success=true, locked=false},
    MigrationScriptProtocol{version=2.1, indexName='null', description='updateMappingOfExistingIndices', scriptName='V002.01__updateMappingOfExistingIndices.http', checksum=787213132, executionTimestamp=2020-11-15T10:54:57.563986+01:00, executionRuntimeInMillis=94, success=true, locked=false},
    MigrationScriptProtocol{version=2.2, indexName='null', description='moveFieldToSearchable', scriptName='V002.02__moveFieldToSearchable.http', checksum=-644953840, executionTimestamp=2020-11-15T10:54:58.189490+01:00, executionRuntimeInMillis=601, success=true, locked=false},
    MigrationScriptProtocol{version=3, indexName='null', description='addDocumentWithNewStructure', scriptName='V003.00__addDocumentWithNewStructure.Http', checksum=1643942673, executionTimestamp=2020-11-15T10:54:58.314287+01:00, executionRuntimeInMillis=100, success=true, locked=false},
    MigrationScriptProtocol{version=3.1, indexName='null', description='addDocumentWithNewStructureToNewIndex', scriptName='V003.01__addDocumentWithNewStructureToNewIndex.http', checksum=1130329556, executionTimestamp=2020-11-15T10:54:58.618791+01:00, executionRuntimeInMillis=276, success=true, locked=false}]>
at ElasticsearchEvolutionIT.lambda$migrate_OK$0(ElasticsearchEvolutionIT.java:66)
[INFO] 
[ERROR] Tests run: 132, Failures: 6, Errors: 0, Skipped: 0
`

Reply:
You're right! Than it's totally OK how you've done it :)